src/components/Avatar.js

```diff
@@ -89,8 +102,11 @@ class Avatar extends PureComponent {
                                 src={this.state.imageError ? this.props.fallbackIcon : this.props.source}
                                 height={iconSize}
                                 width={iconSize}
-                                fill={this.state.imageError ? themeColors.offline : this.props.fill}
-                                isSVGAvatar
+                                fill={this.state.imageError ? themeColors.offline : iconFillColor}
+                                additionalStyles={[
+                                    StyleUtils.getAvatarBorderStyle(this.props.size, this.props.type),
+                                    isWorkspace ? StyleUtils.getDefaultWorspaceAvatarColor(this.props.name) : {},
+                                ]}
```

Comment:
The image error only applies if a workspace has uploaded a png avatar themselves, so I'm not sure which would be expected behavior from a user-perspective. I could follow the default avatar color or I could default to a colorful icon.

<img width="500" alt="image" src="https://user-images.githubusercontent.com/38015950/221986437-fb4636a0-ae20-4492-87bc-00ed233d3349.png"><img width="500" alt="image" src="https://user-images.githubusercontent.com/38015950/221987101-41005c35-6b57-4fc6-b308-0b66712c0bc3.png">

I think I prefer standardizing it to the first option, what do you think?

Reply:
<img width="1681" alt="image" src="https://user-images.githubusercontent.com/38015950/221991146-b9889921-01b5-41a5-a23c-10702a70a228.png">

Or a happy medium?

Reply:
Also, @shawnborton just want to gut check, I noticed these default avatars don't match a color in figma [here](https://www.figma.com/file/OgM6vzR98Qnr1gdmZOQPKz/Implement-Our-New-Brand?node-id=429%3A11870&t=W2UrvAc5GhFRrzGF-0): 

Currently filled with `#1B5744` (and `#8B9C8F`/ icons)

Reply:
Any of the options you proposed work for me

Reply:
Correct @grgia - we picked a different green for the offline fallback so they would be different from the default icon color.

Reply:
Hmm If I use the existing color, I'll need to add it to the theme. Alternatively, I can use the default button color that already exists:

| Using Existing color `#1B5744` | Using Default Button BG `#184E3D` |
| ---- | ---- |
<img width="600" alt="image" src="https://user-images.githubusercontent.com/38015950/222008049-df6e4f71-7535-4956-8076-4d8e04e79fd7.png">|<img width="600" alt="image" src="https://user-images.githubusercontent.com/38015950/222007888-0811ea19-c1ef-488e-b52a-7bd37f9bd7ce.png">|

Reply:
Actually ya know what... I think that existing color used to be the old button color before we darkened up all of the colors as part of the revisions we made. 

So that being said, I am down to use the button color, but that being said, I do think the button color might be the same as the LHN row :hover color so perhaps using the old color is better so we get some separation when the row is :hovered? Or we can pick a new color entirely from our green color stack, maybe like green600 or green700 and see what happens? If we do that, I would want to update all of the old fallback avatars (happy to provide new svgs if needed).

Reply:
Here's what it looks like currently, looks like it's slightly different?

https://user-images.githubusercontent.com/38015950/222245859-0269e69f-cd09-42f1-b2a9-b34e2ff08087.mov

Reply:
I think the current green was from before we did the dark mode revisions, hence why it is different from the new row hover color.

Reply:
Oops sorry I should have specified, this video is using the default button color

Reply:
Oh interesting, maybe that is a buggie then? In Figma, I have it so that the default button color is the same as the border color:

<img width="232" alt="image" src="https://user-images.githubusercontent.com/2319350/222266925-6f1a0ae4-4e0d-42a3-ac28-82e10157d355.png">

Reply:
Ah it must have come from this [one](https://www.figma.com/file/OgM6vzR98Qnr1gdmZOQPKz/Implement-Our-New-Brand?node-id=129%3A4015&t=ILJZwbWl5Rx1Mmaq-0) 
<img width="434" alt="image" src="https://user-images.githubusercontent.com/38015950/222281862-04081ab6-0541-478e-ad65-8a03fd0ef1cb.png">

Reply:
<img width="1202" alt="image" src="https://user-images.githubusercontent.com/38015950/222287377-3d25f0bd-70ee-4e3d-af7d-a3d3b5805a1f.png">

I think it looks good!

Reply:
Same!

Reply:
I'll resolve this, it is looking good now.